website/source/api/secret/identity/group-alias.html.md

```diff
@@ -23,7 +23,7 @@ This endpoint creates or updates a group alias.
   corresponding existing group alias.
 
 - `mount_accessor` `(string: "")` – Mount accessor to which this alias belongs
-  toMount accessor to which this alias belongs to.
+  to.
```

Comment:
Can you remove the first `to` in this sentence?  It reads awkwardly as-is.

Reply:
Ty. Updated